AirCasting/Graph/GraphView.swift

```diff
@@ -22,14 +22,18 @@ struct GraphView<StatsViewModelType>: View where StatsViewModelType: StatisticsC
                               isExpandButtonNeeded: false, isCollapsed: Binding.constant(false),
                               session: session,
                               sessionStopperFactory: sessionStoppableFactory).padding()
-
-            ABMeasurementsView(session: session,
+            
+            ABMeasurementsView(viewModelProvider: {
+                CardMeasurementsViewModel(measurementStreamStorage: measurementStreamStorage,
+                                          sessionDownloader: SessionDownloadService(client: URLSession.shared,
+                                                                                    authorization: UserAuthenticationSession(),
+                                                                                    responseValidator: DefaultHTTPResponseValidator()))
+            },
```

Comment:
yeah, this is also not the place where we should be resolving those dependencies, but that's the best we can do currently without any DI rules and tools in place